### ISSUE TITLE
IniConfigurationSource ignores values with equal sign

### ISSUE THREAD

**Issue by chKarner** [idavis/innovatian.configuration#1]
There is an explicit test case against that:

```
[  NonEmptyAfterCompletelyEmpty   ]
; adding = after comment;=
   mykey    =  myval  akdk     ;
multi= myvalue=do
```

(multi is ignored)
- Is there a reason for that?
- Are there any arguments (Except breaking current behavior) against changing that?

**Comment by idavis** [comment]
I'm sorry, I don't quite understand. Can you please rephrase the issue?

**Comment by chKarner** [comment]
Sorry, ill try again:
When IniConfigurationSource parses data, lines with multiple equal signs(`=`) are skipped.
The code used (in `IniConfigurationSource.GetSection()`) is:

``` c#
string[] pair = workingLine.Split( new[] {Delimiter}, StringSplitOptions.None );
if ( pair.Length != 2 )
{
    continue;
}
```

The test case i cited in the issue description is from the unit test resources, the relevant line for this issue is `multi= myvalue=do`.
The unit tests explicitly wants the line to be ignored, but i'm not sure why (is there some specification for it?)
In my use case i would need the part before the first equal sign be parsed as the key and the rest as the value.

**Comment by idavis** [comment]
Thanks for updating the question. I can't remember why I did it, and I can't think of a good reason for it. I'll update it along with some other features I have had in the backlog.

**Comment by chKarner** [comment]
Great.
I could write the fix myself and open a pull request if youd like.

**Comment by idavis** [comment]
Sure. Thanks!

**Comment by chKarner** [comment]
Fixed by #2.

Are you planning on releasing a nuget package in the near future?

**Comment by idavis** [comment]
I am planning on releasing a new package as soon as I get some feedback on the CustomSettingsClasses branch.
